dataflow/src/ops/base.rs

```diff
@@ -17,6 +19,9 @@ pub struct Base {
     defaults: Vec<DataType>,
     dropped: Vec<usize>,
     unmodified: bool,
+
+    // column_index -> last used auto increment value (starts out as None):
+    auto_increment_values: HashMap<usize, DataType>,
```

Comment:
I think we should require `<= 1` `AUTO_INCREMENT` columns and get rid of the `HashMap`.

Reply:
Fixed 👍